### DISCUSSION TITLE
Animate bootscreen and shutdown screens (like battery charging screen)

### DISCUSSION THREAD

**Discussion by JustinZz7** [OnionUI/Onion#1291] [Ideas]
would like to see animated boot screen and shutdown screen, just like the battery charging screen since we see those screen much more often. Would also speed up the perceived boot time as opposed to looking at a static image

**Comment by Aemiii91** [comment]
The splash screens are static because the process paints to the display once, then exits, leaving all system resources available for the background processes.

This is necessary when working with such low-powered CPU and small amount of memory. We wouldn't want the saving splash to actually cause your save to fail because it runs out of memory (for example).
